app/src/main/java/com/github/polypoly/app/ui/theme/UIElements.kt

```diff
@@ -20,4 +28,108 @@ object UIElements {
             unfocusedLabelColor = MaterialTheme.colors.secondary
         )
     }
+
+    // ================== Buttons ================== //
+
+    /**
+     * Discreet button, use for non-main actions
+     * @param onClick the action to perform when the button is clicked
+     * @param text the text to display on the button
+     * @param testTag the test tag of the button
+     * @param width the width of the button in dp (default 200)
+     * @param height the height of the button in dp (default 60)
+     */
+    @Composable
+    fun DiscreetButton(onClick: () -> Unit, text: String, testTag: String,
```

Comment:
`DiscreetButton` is not a very explicit name, maybe change it to `SecondaryActionButton` or `SecondaryButton`?

Reply:
Yes, I change the name